components/Buttons/src/ColorThemer/MDCButtonColorThemer.h

```diff
@@ -47,7 +47,7 @@
  @param raisedButton An MDCRaisedButton instance to apply a color scheme.
  */
 + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
-                  toRaisedButton:(nonnull MDCRaisedButton *)raisedButton;
+                  toRaisedButton:(nonnull MDCButton *)raisedButton;
```

Comment:
Break this out to a separate change.

Reply:
Done. https://github.com/material-components/material-components-ios/pull/3382